### ISSUE TITLE
Integration of all properties of a product in core properties

### ISSUE THREAD

**Issue by marcofranke** [nimble-platform/common#9]
The index (core) properties defines a set of properties for each concept. The given set of properties doesn't contain properties in which the name of the property is set during the product upload. Examples are e.g.:

1. The user can set the label "width" and a value for the property dimension of a product. 
2. Custom properties have  a label and a value which can be set

The catalogue-search calls this kind of properties "virtual proeprties", because the property in the first example would be not dimension but width and would have a specific value.

These kinds of properties and the properties of eclass are not contained yet. For a semantic (hierachical search) the are needed.

**Comment by suatgonul** [comment]
We will have an additional index field named item_properties including the names of the custom properties via an LDPath expression similar as follows:

item_properties = cac:GoodsItem / cac:Item / cac:AdditionalItemProperty[cbc:ItemClassificationCode / cbc:listID is "Custom"] / cbc:Name :: xsd:string;

**Comment by marcofranke** [comment]
I think , this is a good starting point. With respect to your last ontology, there are more than just custom properties which have to be considered. I mentioned in my issue description also dimension. ( <urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2#Dimension>). Are going to add it also?

**Comment by iot365** [comment]
Yes, the idea is to have an "or" between the "custom" and the "standard" properties. I'm just trying to add such index in the staging environment.
